app/src/pages/AboutUsPage.tsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import styles from "./LandingPage.module.scss";
+import Grid from "@material-ui/core/Grid";
+import Typography from "@material-ui/core/Typography";
+import Accordion from "@material-ui/core/Accordion";
+import AccordionSummary from "@material-ui/core/AccordionSummary";
+import AccordionDetails from "@material-ui/core/AccordionDetails";
+import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
+
+function AboutUsPage() {
+  return (
+    <div className={styles.LandingPage}>
+      <Grid container justify="space-around" className={styles.Grid}>
+        <Grid item xs={12} className={styles.GreenLandingSquare}>
+          <Typography variant="h6" className={styles.HomeLogo}>
+            Contact Info
+          </Typography>
+          <ul>
+            <li>Full address of SHLI</li>
+            <li>Operational hours of SHLI</li>
+            <li>503c identification (need to confirm)</li>
+          </ul>
+        </Grid>
+        <Grid item xs={12} sm={10} md={12}>
+          <Grid item xs={12} className={styles.BlueLandingSquare}>
+            <Typography variant="h6" className={styles.HomeLogo}>
+              FAQ
+            </Typography>
+          </Grid>
+          <Accordion>
+            <AccordionSummary
```

Comment:
nit: I noticed that when these expand, a scrollbar pops up that shifts the content. Could we try something like this: https://css-tricks.com/elegant-fix-jumping-scrollbar-issue/ to make that stop?

Reply:
I think the solution that puts a scroll bar even when one isn't required isn't really worth it (looks kinda tacky to me) and the other one isn't working (creates a gap on the left). I'm inclined to punt this issue until we have more clarity on what the content will actually be since we may not even have the expanding accordions in final version